Python/structmember.c

```diff
@@ -115,6 +120,48 @@ PyMember_GetOne(const char *obj_addr, PyMemberDef *l)
         // doesn't require free-threading code path
         v = Py_NewRef(Py_None);
         break;
+    case Py_T_INT8:
+    case Py_T_INT8|Py_T_UINT8:
```

Comment:
What is this type? I don't understand the usage.

Reply:
It accepts full range of Py_T_UINT8 plus negative values.

Reply:
That's weird. Which existing code needs such type?